UNRELEASED.md

```diff
@@ -7,6 +7,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Enhancements
 
 - Moved icons to a separate npm package ([#686](https://github.com/Shopify/polaris-react/pull/686))
+- Updated the `Page.Header` so that actions take up less room on small screens ([#707](https://github.com/Shopify/polaris-react/pull/707))
```

Comment:
```suggestion
- Updated the `Page` so that actions take up less room on small screens ([#707](https://github.com/Shopify/polaris-react/pull/707))
```

I would just say `Page` because for a consumer that is the only API they know